catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/converter/TestConvertUtil.java

```diff
@@ -252,6 +252,37 @@ public void testFormIcebergType() {
     Assertions.assertTrue(
         ((com.datastrato.gravitino.rel.types.Types.ListType) gravitinoListType).elementType()
             instanceof com.datastrato.gravitino.rel.types.Types.StringType);
+
+    Types.StructType structTypeInside =
```

Comment:
besides type check, could you add the check for other attributes like nullable, name, etc?

Reply:
OK, I'll add it